subsys/net/lib/http/http_server_http1.c

```diff
@@ -5,6 +5,9 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Usually there is no need to mention the path in the commit subject i.e., the `subsys` is kind of redundant information.

Reply:
You are right :)